src/lib/hash/hash.cpp

```diff
@@ -30,6 +30,10 @@
   #include <botan/internal/keccak.h>
 #endif
 
+#if defined(BOTAN_HAS_KECCAK_FIPS)
+  #include <botan/internal/keccak_fips.h>
+#endif
+
```

Comment:
I believe that can go, after `Keccak_FIPS` is not exposed in the public interface.

Reply:
Removed in the last commit.